wondrous-app/components/Common/BountyBoard/index.tsx

```diff
@@ -70,108 +70,111 @@ export default function Board({ tasks, handleCardClick = (bounty) => {}, display
 
   return (
     <>
-      {tasks.map((bounty) => {
-        const reward = bounty?.rewards?.[0];
-        let BountyStatusIcon = TASK_ICONS[bounty?.status];
-        return (
-          <BountyCardWrapper onClick={() => handleCardClick(bounty)} key={bounty.id}>
-            <BoardsCardHeader>
-              <BoardsCardSubheader>
-                <BountyIcon />
-                <BountyCardType>Bounty</BountyCardType>
-                <BoardsPrivacyLabel>
-                  {bounty?.privacyLevel === PRIVACY_LEVEL.public ? 'Public' : 'Members'}
-                </BoardsPrivacyLabel>
-              </BoardsCardSubheader>
-              {bounty?.status === TASK_STATUS_DONE && !bounty?.rewards && <CompletedIcon />}
-              {bounty?.rewards && bounty?.rewards?.length > 0 && (
-                <BoardsRewardLabel>
-                  <Compensation rewards={bounty?.rewards} taskIcon={<BountyStatusIcon />} />
-                </BoardsRewardLabel>
-              )}
-            </BoardsCardHeader>
-            <BoardsCardBody>
-              <BoardsCardBodyTitle>{bounty.title}</BoardsCardBodyTitle>
-              <BoardsCardBodyDescription>
-                <RichTextViewer text={bounty.description} />
-              </BoardsCardBodyDescription>
-              {bounty?.media?.[0] ? (
-                <BoardsCardMedia>
-                  <SafeImage
-                    style={{ width: '100%', height: '100%', objectFit: 'cover', objectPosition: 'center' }}
-                    src={bounty?.media[0].slug}
-                  />
-                </BoardsCardMedia>
-              ) : null}
-              <SubmissionsCount total={bounty.totalSubmissionsCount} approved={bounty.approvedSubmissionsCount} />
-            </BoardsCardBody>
-            <BoardsCardFooter>
-              {bounty?.podName && !displayOrg && (
-                <PodWrapper
-                  style={{ marginTop: '0' }}
-                  onClick={(e) => {
-                    e.preventDefault();
-                    e.stopPropagation();
-                    goToPod(bounty?.podId);
-                  }}
-                >
-                  <PodIcon
-                    color={bounty?.podColor}
-                    style={{
-                      width: '26px',
-                      height: '26px',
-                      marginRight: '8px',
-                    }}
-                  />
-                  <PodName>{bounty?.podName}</PodName>
-                </PodWrapper>
-              )}
-              {displayOrg && (
-                <PodWrapper
-                  style={{ marginTop: '0', alignItems: 'center' }}
-                  onClick={(e) => {
-                    e.preventDefault();
-                    e.stopPropagation();
-                    goToOrg(bounty?.orgId);
-                  }}
-                >
-                  {bounty?.orgProfilePicture ? (
+      {tasks?.length ? (
+        tasks.map((bounty) => {
+          let BountyStatusIcon = TASK_ICONS[bounty?.status];
```

Comment:
you don't overwrite `BountyStatusIcon`, so can use `const` instead

Reply:
done